docs/serving/autoscaling/autoscaling-concepts.md

```diff
@@ -11,7 +11,9 @@ This section covers conceptual information about which Autoscaler types are supp
 
 Knative Serving supports the implementation of Knative Pod Autoscaler (KPA) and Kubernetes' Horizontal Pod Autoscaler (HPA). The features and limitations of each of these Autoscalers are listed below.
 
-**IMPORTANT:** If you want to use Kubernetes Horizontal Pod Autoscaler (HPA), you must install it after you install [Knative Serving](../../install/any-kubernetes-cluster.md#optional-serving-extensions).
+**IMPORTANT:** If you want to use Kubernetes Horizontal Pod Autoscaler (HPA),
+you must install it after you install Knative Serving.
+For how to install HPA, see [Install optional Eventing extensions](../../install/install-extensions.md#install-optional-serving-extensions).
```

Comment:
same comment as above, I'd prefer we use minimalism principles and don't add extra lines that don't add value

Reply:
see my above comment about cross ref format